community/netsurf/depends

```diff
@@ -0,0 +1,6 @@
+curl
+gtk+2
+vim vim
```

Comment:
`vim make`

Adding `make` after a dependency specifies it as a build and not runtime dependency. :)

Reply:
oh finally get it now. I read the website section 3 or 4 times, too, maybe it should be reworded, or maybe I need more sleep.